lib/generators/rollbar/templates/initializer.rb

```diff
@@ -1,4 +1,3 @@
-require 'rollbar/rails'
```

Comment:
Will customers that already have this line in their generated initializer need to remove it? If they don't remove it, what will happen?